header.php

```diff
@@ -21,7 +21,7 @@
 <div id="page" class="site">
 	<a class="skip-link screen-reader-text" href="#content"><?php _e( 'Skip to content', 'newspack' ); ?></a>
 
-	<header id="masthead" class="site-header">
+	<header id="masthead" class="site-header hide-header-search" [class]="searchVisible ? 'show-header-search' : 'hide-header-search'">
```

Comment:
When the toggle occurs this element loses the `site-header` class. To see this in action, use a header with a background and then toggle the search. You should lose the background color the first time you click. I'm not sure this proposal is the most elegant solution, but it will work for now and we can audit all unusual AMP bits later:

```suggestion
	<header id="masthead" class="site-header hide-header-search" [class]="searchVisible ? 'site-header show-header-search' : 'site-header hide-header-search'">
```

Reply:
Oh shoot, I totally missed that! Good work around -- another option may be to move the toggled class to another element.